### PR TITLE
First article in contributor stories series

### PR DESCRIPTION
**Pull request by mbbroberg**
Draft is in @mrbobbytables!

/area contributor-comms
/sig contributor-experience

### CODE REVIEW

File: content/en/blog/2021/contributor-story.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+The Kubernetes contributor community is vast. We have tens of thousands of members by any conservative estimate and many hundreds involved day-to-day. We are from all around the world, all kinds of backgrounds, all types of expertise, and we are all Kubernetes contributors.
+
+We also have this amazing new website dedicated specifically to Kubernetes contributors. With that in mind, let's combine the two. 
+
+We are asking any and **all contributors to share their unique story** to highlight the many ways they show up to be part of the Kubernetes community. Here are some 
+
+## How can I share my contributor story? 
+
+Contributor stories are welcome at this domain, kubernetes.dev, following our regular [blogging practices](https://github.com/kubernetes/community/blob/4026287dc3a2d16762353b62ca2fe4b80682960a/sig-docs/blog-subproject/README.md#submit-a-post). The main change is in the location: pull requests can be directed to the [contributor site repository](https://github.com/kubernetes/contributor-site) for this series.
```
Comment (mrbobbytables) [inline content/en/blog/2021/contributor-story.md:18]:
```suggestion
Contributor stories are welcome at this domain, kubernetes.dev, following our regular [blogging practices](https://github.com/kubernetes/community/blob/master/sig-docs/blog-subproject/README.md#submit-a-post). The main change is in the location: pull requests can be directed to the [contributor site repository](https://github.com/kubernetes/contributor-site) for this series.
```

File: content/en/blog/2021/contributor-story.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+title: "Announcing the Kubernetes Contributor Stories Series"
+linkTitle: "Share your Kubernetes Contributor Story"
+date: 2021-04-11
```
Comment (mrbobbytables) [inline content/en/blog/2021/contributor-story.md:4]:
```suggestion
date: 2021-04-21
```
I just randomly picked the date, but it needs to be bumped to match when this is intended to be released.

**Review by mrbobbytables — COMMENTED** [review]
Thanks!
Two small changes, but I think that should cover it :)

/assign @parispittman

**Comment by mbbroberg** [comment]
All tidied up and ready for your approval! Check it out @mrbobbytables @chris-short @Pensu 👍

**Pull request merged**